test-harness.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
FYI - I've made various changes to the original script based on feedback. The biggest change is [defaulting to not overwriting testing SDK's .env](https://github.com/algorand/py-algorand-sdk/blob/trim-indexer/test-harness.sh#L30-L33)

Reply:
done!